### PR TITLE
planeta.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: planeta.ru.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<ruleset name="planeta.ru">
+	<target host="planeta.ru" />
+	<target host="www.planeta.ru" />
+	<target host="biblio.planeta.ru" />
+	<target host="s2.planeta.ru" />
+	<target host="school.planeta.ru" />
+	<target host="shop.planeta.ru" />
+	<target host="static.planeta.ru" />
+	<target host="tv.planeta.ru" />
+	<target host="files.planeta.ru" />
```
Comment (J0WI) [inline planeta.ru.xml]: cert expired

File: planeta.ru.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	<target host="" />
```
Comment (J0WI) [inline planeta.ru.xml]: empty target

File: planeta.ru.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<!--
+
+
+-->
+<ruleset name="planeta.ru">
+	<target host="planeta.ru" />
+	<target host="www.planeta.ru" />
```
Comment (J0WI) [inline planeta.ru.xml]: mismatching cert

**Comment by J0WI** [comment]
I wonder why the tests passed with all these failures.
Let me just reopen to check again.

**Comment by J0WI** [comment]
It seems that [this](https://travis-ci.org/EFForg/https-everywhere/jobs/158452399) is not checked because of `Core code changes have been made.` cc @Hainish

**Comment by J0WI** [comment]
Now this rule has no targets at all.

**Comment by galeksandrp** [comment]
Fixed.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**